### PR TITLE
Add 7guis Readme

### PR DESCRIPTION
**Pull request by lukas-jung**
I guess you want to move the images to the server. Those are just here for the Draft.

### CODE REVIEW

**Comment by tronical** [comment]
> I guess you want to move the images to the server. Those are just here for the Draft.

I'm curious, why move them to the web server instead of leaving them in here? (maybe in a sub-directory)

**Comment by tronical** [comment]
Not necessarily applicable for this change, but I was positively surprised to see this: https://twitter.com/peha23/status/1525018586854612993 . This suggests that in the future we could have a "update screenshots" GitHub action that runs various examples, generates screenshots and makes a merge request out of them.

**Comment by lukas-jung** [comment]
Well all the other screenshots are on your server. That's why i thought you preferred it that way.
Also then users don't always copy the screenshots to their drive if they clone it.

File: examples/7guis/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Slint's 7guis implementation
+
+[`7guis`](https://eugenkiss.github.io/7guis/) is a "GUI Programming Benchmark".
```
Comment (ogoffart) [inline examples/7guis/README.md:3]:
```suggestion
[7GUIs](https://eugenkiss.github.io/7guis/) is a "GUI Programming Benchmark".
```
(This is a title, not code)

**Comment by lukas-jung** [comment]
Also the CI doesn't like the images, because they don't have a license..

**Comment by tronical** [comment]
> Also then users don't always copy the screenshots to their drive if they clone it.

That's a good argument - thanks for pointing it out :)

I don't have a strong feeling either way.

**Comment by ogoffart** [comment]
> Well all the other screenshots are on your server.

They used to be in this repository.
But we moved them to https://github.com/slint-ui/website/tree/master/data/resources , i don't remember the reason.
Commit c52c8ad664d5e58fa7518e449e45a31b72750278 gives a reason but i'm not sure i understand.

I guess it makes sense to have the screenshots on the website repo, although having them in this repo might be fine.

File: examples/7guis/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Just a Button that increases a value in a text field.
+
+![Screenshot of the 7guis Counter](counter.png "Counter")
+
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/counter.slint)
+
+## [Temperature Converter](https://eugenkiss.github.io/7guis/tasks/#temp)
+Converts Celsius to Fahrenheit and vice versa.
+
+![Screenshot of the 7guis Temperature Converter](tempconv.png "Temperature Converter")
```
Comment (ogoffart) [inline examples/7guis/README.md:18]: I wonder if it wouldn't look better without the explicit preferred-height and letting the line edit be the height of a normal line edit

File: examples/7guis/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/tempconv.slint)
+
+## [Flight Booker](https://eugenkiss.github.io/7guis/tasks/#flight)
+Performs some validation checking on dates.
+Does not actually book flights.
+
+![Screenshot of the 7guis Flight Booker](booker.png "Flight Booker")
+
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/booker.slint)
+(Not functional on its own)
```
Comment (ogoffart) [inline examples/7guis/README.md:29]: maybe explain why it is not "fonctional on its own"?

```suggestion
(Not functional on its own, logic in the Rust file)
```

File: examples/7guis/README.md (hunk 2 above)
Comment (lukas-jung) [reply examples/7guis/README.md:18]: I wanted to do this. But I thought it caused weird resizing behavior with preview.
But actually resizing preview windows is broken either way...
At least on my system. The calculated minimum window size is wrong and it kind of glitches when you make it smaller.

Comment (lukas-jung) [reply examples/7guis/README.md:18]: I opened it here: #1269

File: examples/7guis/README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/tempconv.slint)
+
+## [Flight Booker](https://eugenkiss.github.io/7guis/tasks/#flight)
+Performs some validation checking on dates.
+Does not actually book flights.
+
+![Screenshot of the 7GUIs Flight Booker](https://user-images.githubusercontent.com/22800467/168557449-769df1cd-f967-4e14-bc5c-d8eeccc33305.png "Flight Booker")
+
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/booker.slint)
+(Not functional on its own, logic in the Rust file)
```
Comment (tronical) [inline examples/7guis/README.md:29]:
```suggestion
Note that the logic for date validation and date comparison is implemented in [Rust](./booker.rs).
```

File: examples/7guis/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+## [Timer](https://eugenkiss.github.io/7guis/tasks/#timer)
+A simple timer. The time is adjustable while running.
```
Comment (tronical) [inline examples/7guis/README.md:32]:
```suggestion
A simple timer where the duration is adjustable while running.
```

File: examples/7guis/README.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+![Screenshot of the 7GUIs Timer](https://user-images.githubusercontent.com/22800467/168557131-68382191-9228-4d58-9683-6648ab5e7efd.png "Timer")
+
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/timer.slint)
+(Not functional on its own, logic in the Rust file)
```
Comment (tronical) [inline examples/7guis/README.md:37]:
```suggestion
Note that the actual timer functionality is implemented in [Rust](./timer.rs).
```

File: examples/7guis/README.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+## [CRUD](https://eugenkiss.github.io/7guis/tasks/#crud)
+Lets you create, read, update and delete names from a list as well as filter them by prefix.
+Our implementation makes use of `MapModel` and `FilterModel` to achieve this.
+
+![Screenshot of the 7GUIs CRUD](https://user-images.githubusercontent.com/22800467/168557502-93c87141-3eb5-410c-9b83-4b7342727e37.png "CRUD")
+
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/crud.slint)
+(Not functional on its own, logic in the Rust file)
```
Comment (tronical) [inline examples/7guis/README.md:46]:
```suggestion
Note that the changes to the data model and filtering is implemented in [Rust](./crud.rs).
```

File: examples/7guis/README.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+## [Circle Drawer](https://eugenkiss.github.io/7guis/tasks/#circle)
+Draw some circles on a canvas and change their sizes. It has undo and redo capabilities.
+
+![Screenshot of the 7GUIs Circle Drawer](https://user-images.githubusercontent.com/22800467/168557533-7632efba-3b3b-459d-a8c0-6f166fa42e23.png "Circle Drawer")
+
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/circledraw.slint)
+(Not functional on its own, logic in the Rust file)
```
Comment (tronical) [inline examples/7guis/README.md:54]:
```suggestion
Note that the undo-redo stack is implemented in [Rust](./circledraw.rs).
```

File: examples/7guis/README.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+## [Cells](https://eugenkiss.github.io/7guis/tasks/#cells)
+Almost MS Excel. It uses nested models to create the table.
+
+![Screenshot of the 7GUIs Cells](https://user-images.githubusercontent.com/22800467/168557595-95ad3255-006c-416a-bccd-8f5251adebd7.png "Cells")
+
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/cells.slint)
+(Not functional on its own, logic in the Rust file)
```
Comment (tronical) [inline examples/7guis/README.md:62]:
```suggestion
Note that the cell model, expression evaluation and dependency handling is implemented in [Rust](./cells.rs).
```

**Review by tronical — APPROVED** [review]

**Pull request merged**